src/Adapter/Order/QueryHandler/GetOrderForViewingHandler.php

```diff
@@ -446,7 +449,10 @@ private function getOrderProducts(Order $order): OrderProductsForViewing
                 $product['current_stock'],
                 $imagePath,
                 Tools::ps_round($product['unit_price_tax_excl'], 2),
-                Tools::ps_round($product['unit_price_tax_incl'], 2)
+                Tools::ps_round($product['unit_price_tax_incl'], 2),
```

Comment:
Same feedback as for Franck's PR I'm not sure it's a good idea to round values like this, either not round at all Or if we round them we should use ComputePrecision and the appropriate currency precision I believe he has some modifications to make in the same handler in https://github.com/PrestaShop/PrestaShop/pull/16150

Reply:
done !